packages/salesforcedx-vscode-apex/package.json

```diff
@@ -225,6 +226,13 @@
           ],
           "default": false,
           "description": "%apex_semantic_errors_description%"
+        },
+        "salesforcedx-vscode-apex.enable-sobject-refresh-on-startup": {
+          "type": [
+            "boolean"
+          ],
+          "default": true,
+          "description": "%enable_sobject_refresh_on_startup_description%"
```

Comment:
The `sfdx.force.internal.refreshsobjects` commands from the core extension's `package.json` need to be moved to this `package.json`